diff --git a/board/helipilot/build.mk b/board/helipilot/build.mk
index 23b1945fed..6ccb2d6613 100644
--- a/board/helipilot/build.mk
+++ b/board/helipilot/build.mk
@@ -30,7 +30,6 @@ test-list-y=\
        benchmark \
        boringssl_crypto \
        compile_time_macros \
-       cortexm_fpu \
        crc \
        debug \
        exception \
@@ -64,7 +63,6 @@ test-list-y=\
        static_if \
        stdlib \
        std_vector \
-       stm32f_rtc \
        system_is_locked \
        timer \
        timer_dos \
