diff --git a/zephyr/program/fpmcu/et171/prj.conf b/zephyr/program/fpmcu/et171/prj.conf
index dbe0e97671..a7cff1ab13 100644
--- a/zephyr/program/fpmcu/et171/prj.conf
+++ b/zephyr/program/fpmcu/et171/prj.conf
@@ -1,3 +1,5 @@
+CONFIG_PLATFORM_EC_DEBUG_ASSERT=n
+
 # Copyright 2025 The ChromiumOS Authors
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
